### PR TITLE
Pointers

### PR DESCRIPTION
**Pull request by Manithj**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# learningC-
\ No newline at end of file
+# learning_Cpp
+
+C++ is a powerful general-purpose programming language. It can be used to develop operating systems, browsers, games, and so on. C++ supports different ways of programming like procedural, object-oriented, functional, and so on. This makes C++ powerful as well as flexible.
+
+- [Pointers](https://github.com/Manithj/learning_Cpp/tree/pointers/Pointers)
```
Comment (PramudithaPothuwila) [inline README.md]: Make the path relative

**Review by PramudithaPothuwila — CHANGES_REQUESTED** [review]
Change the paths to relative in the main readme file

**Comment by Manithj** [comment]
Pointers path in the main readme file?

**Comment by Manithj** [comment]
Done

**Review by PramudithaPothuwila — APPROVED** [review]

**Pull request merged**